### DISCUSSION TITLE
my account flagged i cant reac my repositories , i will lost my work

### DISCUSSION THREAD

**Discussion by Budalebah** [community/community#135817] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

after using vercel my account was flagged as flagged. I can't access my projects and I can't work. I created a support ticket but I didn't get a response. I will lose my job for this reason, please help.
Username : budalebah

**Comment by AnouarMohamed** [comment]
This is unfortunately a known issue where automated spam filters occasionally flag accounts during Vercel integration if the project setup triggers certain 'high-activity' patterns.

Since you’ve already opened a ticket, the best thing to do is Reply to the automated confirmation email you received. This 'bumps' the ticket in their internal queue. Additionally, ensure you haven't recently changed your primary email or payment method on Vercel, as that can sometimes trigger the flag. GitHub Support is the only one who can lift the flag manually, but replying to the ticket usually speeds up the process.
